apps/protocol-frontend/src/contexts/UserContext.tsx

```diff
@@ -230,6 +230,12 @@ export const UserContextProvider: React.FC<UserContextProps> = ({
     }
   }, [userDataByAddress, isAuthenticated]);
 
+  // useEffect(() => {
+  //   if (userDataByAddress && isAuthenticated) {
+  //     getUser();
+  //   }
+  // }, [userData?.guild_users, isAuthenticated, userDataByAddress]);
+
```

Comment:
This should probably be commented back in

Reply:
Do we want to add this back in now or wait until we have the more robust UserContext refactor and use that instead?

Reply:
Issue with this that I'm seeing -- I split this into it's own effect to avoid the circular dependencies,  but then we'd introduce another lint warning (and another useEffect)

We could do that or refactor to use react query for the user DAO's check

Reply:
If it is commented out we want to remove it

Reply:
Should be gone, let me double check if it didn't make the commit for some reason